atlp-rwanda/commanders-medica-backoffice
# Feature: dashboard login and signup functionality

#### What does this PR do?

This PR includes designs and implantation of login and signup features.

#### Description of Task to be completed?

- Design login and signin pages
- Design guest navbar
- Implement signup and login functionalities

#### How should this be manually tested?

Start the app and click on login link on the navbar to check the login page. For signup page click on signup.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
